### PR TITLE
Populate bbn-test-5 network page

### PR DESCRIPTION
**Pull request by filippos47**
Made successful test boot.

Pending actions:
- ~~Collect all the node operator endpoints~~
- Pinpoint Babylon version

### CODE REVIEW

File: bbn-test-5/README.md [hunk 1]
```diff
@@ -8,17 +8,62 @@ in the covenant committee.
...
+
+<!-- TODO:
+- Nodes.Guru endpoint
+- Polkachu endpoint
+- In-house endpoint (in case we decide to include)
+-->
+
+### Endpoints
+
+#### RPC
```
Comment (vitsalis) [inline bbn-test-5/README.md:33]: A bit difficult to differentiate between level-3 and level-4 headings. How about we either add numbering to the sections or use a different type of heading for RPC?

File: bbn-test-5/README.md [hunk 2]
```diff
@@ -8,17 +8,62 @@ in the covenant committee.
...
+#### Archive nodes
+
+<!-- TODO:
+- Nodes.Guru endpoint
+- Polkachu endpoint
+-->
+
+### State snapshot
+
+The following snapshot includes state for blocks `1` to `192`, which have
```
Comment (vitsalis) [inline bbn-test-5/README.md:63]: Is it typical to say the range of block heights?

Also the `which have already been produced` sentence is redundant.

File: bbn-test-5/README.md [hunk 3]
```diff
@@ -8,17 +8,62 @@ in the covenant committee.
...
+already been produced:
+
+- [Testnet-5 State snapshot](./network-artifacts/bbn-test-5.tar.gz)
```
Comment (vitsalis) [inline bbn-test-5/README.md:66]: Are we planning to have more than one snapshot and including a list here?

File: bbn-test-5/README.md (hunk 2 above)
Comment (filippos47) [reply bbn-test-5/README.md:63]: I see Polkachu mentions directly the upper limit. We can do the same.

File: bbn-test-5/README.md (hunk 3 above)
Comment (filippos47) [reply bbn-test-5/README.md:66]: Nope, removing the list.

File: bbn-test-5/README.md (hunk 1 above)
Comment (filippos47) [reply bbn-test-5/README.md]: Added numbering.

**Review by vitsalis — APPROVED** [review]

**Review by vitsalis — APPROVED** [review]

**Pull request merged**